.dev-lib

```diff
@@ -1,8 +1,8 @@
 DEFAULT_BASE_BRANCH=develop
 PHPCS_GIT_TREE=master
```

Comment:
As Composer is being used now, I would presume that we can drop the `PHPCS_GIT_TREE` and `WPCS_GIT_TREE` environment variables  in the `.dev-lib` file ?